x-pack/test/functional/services/ml/navigation.ts

```diff
@@ -36,6 +36,11 @@ export function MachineLearningNavigationProvider({
       });
     },
 
+    async navigateToAlertsAndAction() {
+      await PageObjects.common.navigateToApp('triggersActions');
+      await testSubjects.click('alertsTab');
```

Comment:
Would be good to assert that the expected page / container is displayed after this click.

Reply:
Added in https://github.com/elastic/kibana/pull/89286/commits/3e569c79418823882d7892129037866ac5c1bf0c